libs/strings/storage_string/README.md

```diff
@@ -0,0 +1,58 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".docs/storage-string-logo-dark-theme.png">
+        <img alt="SwayApps logo" width="400px" src=".docs/storage-string-logo-light-theme.png">
+    </picture>
+</p>
+
+# Overview
+
+The StorageString library provides an interface to store UTF-8 encoded strings of dynamic length in Sway. The `StorageString` can be used in combination with the `String` type.
+
+The `StorageString` is stored as a collection of tightly packed bytes. This differs from Sway's built in `str` because the size cannot be known at compile time and the length is dynamic. 
+
+For more information please see the [specification](./SPECIFICATION.md).
+
+## Known Issues
+
+Until https://github.com/FuelLabs/fuels-rs/issues/940 is resolved, developers must use the `Bytes` type to return a `String` from a contract.
+
+# Using the Library
+
+## Getting Started
+
+In order to use the `StorageString` library it must be added to the Forc.toml file and then imported into your Sway project. To add Sway-libs as a dependency to the Forc.toml in your project, please see the [README.md](../../../README.md).
+
+```rust
+use storage_string::StorageString;
+```
+
+Once imported, a `StorageString` must be defined in a storage block.
+
+```rust
+storage {
+    stored_string: StorageString = StorageString {},
+}
+```
+
+## Basic Functionality
+
+Storing a `String` type can be done by calling the `store` function.
+
+```rust
+// Create a new string
+let mut my_string = String::new();
+my_string.push(0u8);
```

Comment:
For this basic example, could this push something other than an integer?
i.e: can "Hello" be pushed, or do we need to manually translate a string to bytes encoding to use this?

Reply:
We would manually need to translate this to bytes unfortunately. This is where we are lacking in terms of `str` and `String` conversion.

Reply:
No worries; I review a little further and saw where you make this conversion in the SDK tests.